packages/authentication/package.json

```diff
@@ -39,6 +39,7 @@
     "@loopback/core": "^2.13.1",
     "@loopback/eslint-config": "^10.0.4",
     "@loopback/openapi-spec-builder": "^3.0.3",
+    "@loopback/openapi-v3": "^5.1.2",
```

Comment:
Let's use `@loopback/rest` for imports as types in `@loopback/openapi-v3` are re-exported in `@loopback/rest`.

Reply:
Done